pymeasure/instruments/keithley/keithley2600.py

```diff
@@ -86,19 +89,19 @@ def complete(self):
         query places an ASCII character 1 into the device's Output Queue when all pending
         selected device operations have been finished.
         """
-        return self.ask("waitcomplete() print([[1]])").strip()
-
-    @property
-    def status(self):
```

Comment:
Does the `Keithley2600` still have a `status` property?

Reply:
It doesn't - I was thinking the child class should define the `status` property, but I think it makes more sense in the base class (as this won't change), I will move it.

Reply:
I just didn't see it get moved. I think what you've done is fine. If it's truly common, then you're doing coders a favour, but if it will often need to be customized you might be making it less easy for coders to know so.